src/test/java/com/newrelic/experts/jenkins/EventRecorderImplTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.newrelic.experts.jenkins;
+
+import com.newrelic.experts.client.model.Event;
+import com.newrelic.experts.client.model.JenkinsMasterEvent;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+
+
+/**
+ * @author dalvizu
+ */
+@Test(groups = { "unit" })
+public class EventRecorderImplTest {
```

Comment:
@dalvizu appreciate you adding this test.  We can test the `recordJenkinsMasterEvent()` here as well.  Also, on the current `master`, I had to migrate over to JUnit but the changes should be minimal.

Reply:
I've ported this test to JUnit and added mocks around label and hostname methods we query Jenkins on